lib/cloudkeeper/aws/filter_helper.rb

```diff
@@ -0,0 +1,39 @@
+module Cloudkeeper
+  module Aws
+    # Class used for generating filter structures used by AWS .describe_tags
+    class FilterHelper
```

Comment:
Maybe try to refactor the class using the builder pattern so you can write something like `FilterHelper.only.by_name('name')`.